src/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.c

```diff
@@ -25,14 +26,43 @@ ActorProfile Npc_Engineer_Profile = {
 };
 #endif
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer/aNEG_actor_ct.s")
+//#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer/aNEG_actor_ct.s")
+extern struct_809AEFA4 D_809AEFA4_jp;
```

Comment:
Don't forget to migrate the data to the C file. Currently the data is located at `asm/jp/data/overlays/actors/ovl_Npc_Engineer/ac_npc_engineer.data.s`. If you have problems migrating the data feel free to ask